features/customer_booking_request.feature

```diff
@@ -12,3 +12,13 @@ Scenario: Customer browses a regular location
   Given no locations are enabled for online booking
   When I browse for the location
   Then I cannot book online
+
+@vcr
+Scenario: Customer makes an online Booking Request
+  Given a location enabled for online booking
```

Comment:
Not sure I feel that strongly! But equally I don't see how it makes sense as it is currently? 🤔